openstudiocore/src/model/People.cpp

```diff
@@ -423,9 +423,9 @@ namespace detail {
   }
 
   boost::optional<double> People_Impl::spaceFloorAreaPerPerson() const {
-    OptionalDouble temp = peoplePerFloorArea();
+    OptionalDouble temp = peopleDefinition().spaceFloorAreaperPerson();
     if (temp) {
-      return 1.0 / temp.get();
+      return temp.get() * multiplier();
```

Comment:
This should get divided by the load instance multiplier right?

Reply:
@jmarrec if you have 100ft2/person and then you add a multiplier of 5, you want 100ft/(5person) right?  Or do you want 500ft2/person (larger multiplier gets fewer people).  Multipliers are kind of weird with this one

Reply:
That might be why this was the way it was?

Reply:
I agree with you. The space should be 5 times more crowded... We should check if the multiplier is strictly greater than 0 before dividing too.

Reply:
The way it was is wrong though.

peopleDefintion.setSpaceFloorAreaperPerson(10) 
then from the linked people: 
people.spaceFloorAreaPerPerson => I will get a non initialized currently!